Examples/ML/brain.py

```diff
@@ -0,0 +1,20 @@
+from sportsreference.nfl.teams import Teams
+from sklearn.linear_model import LinearRegression
+import numpy as np
+
+starting_year = 1970
+wins = []
+years = []
+ending_year = 2020
+
+for i in range(starting_year, ending_year):
```

Comment:
I would recommend a more descriptive variable name than `i` here as it's generally suggested to not use single-character variables if possible. Perhaps `year` would be good? Just putting this comment here, but it would apply to the variable when used below as well.